app/components/Settings/ConnectButton/ConnectButton.react.js

```diff
@@ -12,6 +22,38 @@ export default class ConnectButton extends Component {
         editMode: PropTypes.bool
     }
 
+    /**
+    * Will check whether connection requests are equal to or greater then 400
+    * @returns {boolean} true if connection error
+    */
+    isConnectionError() {
+        return (this.props.connectRequest.status >= 400 || this.props.saveConnectionsRequest.status >= 400);
+    }
+
+    /**
+     * Checks the connection requests to see if status is loading
+     * @returns {boolean} true if status is loading
+     */
+    loadingStatus() {
+        return (isLoading(this.props.connectRequest.status) || isLoading(this.props.saveConnectionsRequest.status));
```

Comment:
`isLoading` is nw only used here. I'd get rid of `isLoading` and rename `loadingStatus` to `isLoading` (it's confusing to have two function so similar in name and purpose)